### PR TITLE
addons + plans - integration tests

### PR DESCRIPTION
**Pull request by mchalski**
part 2 of:
https://tracker.mender.io/browse/MEN-4510

check feature availability across plans and addons.
exercise plan and addon upgrades as well.
this is to make sure:
- restrictions are in place, so that 'existing' users are incentivized to upgrade
- when we do actually upgrade them - access is granted accordingly

### CODE REVIEW

**Comment by mchalski** [comment]
@mender-test-bot start pipeline

**Comment by mchalski** [comment]
@mender-test-bot start pipeline

File: testutils/api/deviceconnect.py [hunk 1]
```diff
@@ -15,3 +15,4 @@
 URL_MGMT_DEVICE = "/devices/{id}"
 URL_MGMT_CONNECT = "/devices/{id}/connect"
 URL_MGMT_PLAYBACK = "/sessions/{session_id}/playback"
+URL_MGMT_FDOWNLOAD = "/devices/{id}/download"
```
Comment (tranchitella) [inline testutils/api/deviceconnect.py:18]: Shall we test the /upload end-point, as well?

File: testutils/api/tenantadm_v2.py [hunk 2]
```diff
@@ -16,6 +16,8 @@
 URL_MGMT = "/api/management/v2/tenantadm"
 
 URL_CREATE_ORG_TENANT = "/tenants"
+URL_CREATE_ORG_TRIAL = "/tenants/trial"
+URL_TENANT_UPGRADE_START = "/tenants/{id}/upgrade/start"
+URL_TENANT_UPGRADE_COMPLETE = "/tenants/{id}/upgrade/complete"
 URL_TENANT_STATUS = "/tenants/{id}/status"
 URL_TENANT_SECRET = "/billing/secret"
-URL_CONTACT_SUPPORT = "/contact/support"
```
Comment (tranchitella) [inline testutils/api/tenantadm_v2.py:21]: Why removing this line?

**Review by tranchitella — APPROVED** [review]

File: testutils/api/tenantadm_v2.py (hunk 2 above)
Comment (mchalski) [reply testutils/api/tenantadm_v2.py:21]: that's a mistake, thanks

File: testutils/api/deviceconnect.py (hunk 1 above)
Comment (mchalski) [reply testutils/api/deviceconnect.py:18]: the idea was to test each feature very lightly, but sure, it's a cheap addition so I'll add it.

**Comment by mchalski** [comment]
@mender-test-bot start pipeline

**Comment by mchalski** [comment]
@mender-test-bot start pipeline

**Pull request merged**